app/models/review.rb

```diff
@@ -0,0 +1,13 @@
+class Review < ApplicationRecord
+  after_create :update_average_rating
+  belongs_to :restaurant
+
+  validates :reviewer, :rating, :comment, presence: true
+  validates :rating, inclusion: { in: [0, 1, 2, 3] }
```

Comment:
Should probably use `numericality` validation here.

Reply:
This is a weird rating scale